Readme.md

```diff
@@ -553,3 +617,212 @@ Example:
 
 Inserts a no$gba debug message as described by GBATEK.
 
+# 5. Macros
```

Comment:
Thanks, I agree that undocumented features are better than incorrectly documented features. I'll compare the docs against the code and update this.